smoke/IotEdgeQuickstart/details/IotedgedWindows.cs

```diff
@@ -137,23 +174,33 @@ public async Task Configure(string connectionString, string image, string hostna
                 string[] result = await Process.RunAsync("powershell", args, cts.Token);
                 WriteToConsole("Output from Configure iotedge windows service", result);
 
-                UpdateConfigYamlFile(runtimeLogLevel);
+                // Stop service and update config file
+                await this.Stop().ConfigureAwait(false);
+
+                UpdateConfigYamlFile(deviceCaCert, deviceCaPk, deviceCaCerts, runtimeLogLevel);
 
                 // Explicitly set IOTEDGE_HOST environment variable to current process
                 SetEnvironmentVariable();
             }
         }
 
-        static void UpdateConfigYamlFile(LogLevel runtimeLogLevel)
+        static void UpdateConfigYamlFile(string deviceCaCert, string deviceCaPk, string deviceCaCerts, LogLevel runtimeLogLevel)
```

Comment:
Nit: rename deviceCaCerts to ownerCert or TrustBundleCerts or something like that..

Reply:
Updated in another PR.